entries/b/bankera.com.json

```diff
@@ -6,10 +6,14 @@
     ],
     "documentation": "https://bankera.com/faq/how-to-enable-second-factor-authentication/",
     "categories": [
-      "banking"
+      "payments"
     ],
     "regions": [
-      "-us"
+      "-us",
+      "-kp",
+      "-ru",
+      "-ir",
+      "-by"
     ]
```

Comment:
Given that it gives users a European IBAN with SEPA payment support, should we restrict Bankera to EU regions?